src/backend/columnar/Makefile

```diff
@@ -1,10 +1,16 @@
+#SUBDIRS = . safeclib
+#OBJS += \
+	$(patsubst $(citus_abs_srcdir)/%.c,%.o,$(foreach dir,$(SUBDIRS), $(sort $(wildcard $(citus_abs_srcdir)/$(dir)/*.c))))
+
+safestringlib_srcdir = $(citus_abs_top_srcdir)/vendor/safestringlib
 MODULE_big = columnar
-OBJS = mem_primitives_lib.o \
-       memcpy_s.o \
-	   safe_str_constraint.o \
-	   safe_mem_constraint.o \
-	   ignore_handler_s.o \
-	   abort_handler_s.o \
+
+OBJS = safeclib/mem_primitives_lib.o \
+       safeclib/memcpy_s.o \
+	   safeclib/safe_str_constraint.o \
+	   safeclib/safe_mem_constraint.o \
+	   safeclib/ignore_handler_s.o \
+	   safeclib/abort_handler_s.o \
```

Comment:
Do we need all of these to avoid the missing symbol errors? If so that's fine, if not let's remove the ones we don't need.